### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by Lumenbolt** [keepassium/KeePassium#175]
I know this might not be a much-demanded feature, but it would be great to have a simple dark theme as all other apps/password managers have an option for one.

**Also, can we have an heads-up on when we can get a diceware (passphrase) generator (Feature request #78)?
Having passphrases really helps to remember the password temporarily for a particular entry, and especially when one has enter it on a machine in which the user might not have its database on (good luck manually typing a long string of random characters without committing a mistake)**

P.S. absolutely love the project!

**Comment by keepassium** [comment]
> I know this might not be a much-demanded feature, but it would be great to have a simple dark theme as all other apps/password managers have an option for one.

But KeePassium has a dark mode (#35), It follows the system settings (Settings → Display & Brightness → Dark) Or do you mean something else?

> can we have an heads-up on when we can get a diceware (passphrase) generator

Unfortunately, I cannot promise any specific timeline until the feature has been submitted to App Store review. Life happens, and rather than break promises I'd rather not promise at all... For now, I can only say it's on the radar.

**Comment by Lumenbolt** [comment]
> KeePassium has a dark mode (#35)

I see, apparently it's not available in iOS 12.5.4 in which the latest version of Keepassium is 1.24.87. But that's alright, no issues.

> I cannot promise any specific timeline until the feature has been submitted to App Store review

Got it, I think a diceware generator (#78) is a much-needed feature in a password manager nowadays since a lot of us regularly use it for typing the passwords down or to remember it temporarily.

**Comment by keepassium** [comment]
> apparently it's not available in iOS 12.5.4

Yep, system-wide dark mode was introduced only in iOS 13. And with iOS 12 being on less than 8% of devices, it would not make much sense to back-port it...

> I think a diceware generator (#78) is a much-needed feature in a password manager nowadays since a lot of us regularly use it for typing the passwords down or to remember it temporarily.

I agree, it would be useful. But having only 24 hours in a day I have to prioritize :) 

By the way, if you often need to type passwords to a computer, you might want to check the [Input Stick](http://inputstick.com).
